packages/web-frontend/package.json

```
@@ -18,13 +18,16 @@
     "start": "PORT=8088 yarn package:start:react-scripts",
     "start-dev": "yarn start",
     "start-mobile": "REACT_APP_APP_TYPE=mobile PORT=8089 PORT=8088 package:start:react-scripts",
-    "start-fullstack": "npm-run-all -c -l -p start-central-server start-entity-server start-report-server start-web-config-server start-dev",
+    "start-fullstack": "npm-run-all -c -l -p start-servers start-dev",
     "start-central-server": "yarn workspace @tupaia/central-server start-dev",
     "start-entity-server": "yarn workspace @tupaia/entity-server start-dev",
     "start-report-server": "yarn workspace @tupaia/report-server start-dev",
     "start-web-config-server": "yarn workspace @tupaia/web-config-server start-dev",
+    "start-servers": "npm-run-all -c -l -p start-central-server start-entity-server start-report-server start-web-config-server",
     "storybook": "start-storybook -s public -p 6007",
-    "test": "node scripts/test.js --env=jsdom"
+    "test": "node scripts/test.js --env=jsdom",
+    "start-ui-components": "yarn workspace @tupaia/ui-components start-dev",
+    "start-frontend": "npm-run-all -c -l -p start-components start-dev"
```

```suggestion
    "start-frontend": "npm-run-all -c -l -p start-ui-components start-dev"
```
I think this should be start-ui-components right?